### PR TITLE
Updated at 28-01-2020 07:08PM

### PR DESCRIPTION
**Pull request by shubhamvekariya**

### CODE REVIEW

File: OnlineParkingSystem/Landlord/views.py [hunk 1]
```diff
@@ -72,11 +67,10 @@ def AddLandDetail(request):
             land.save()
             return render(request, 'AddLandDetail.html',{'form' : form})
         else:
-            print("=========================Something went wrong=========================")
+            print("dasd")
```
Comment (kevaltalaviya) [inline OnlineParkingSystem/Landlord/views.py]: don't require

Comment (shubhamvekariya) [reply OnlineParkingSystem/Landlord/views.py]: It's done

File: OnlineParkingSystem/User/templates/LandDetails.html [hunk 2]
```diff
@@ -56,9 +56,9 @@
         <td>{{land.price_per_hour}}</td>
         <td>{{land.start_date}}</td>
         <td>{{land.end_date}}</td>
+        <td>{{land.d}}</td>
```
Comment (kevaltalaviya) [inline OnlineParkingSystem/User/templates/LandDetails.html]: give appropriate name

Comment (shubhamvekariya) [reply OnlineParkingSystem/User/templates/LandDetails.html]: It's done

**Pull request merged**